packages/page-treasury/src/Tips/SlashTipModal.tsx

```diff
@@ -0,0 +1,91 @@
+// Copyright 2017-2021 @polkadot/app-treasury authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import BN from 'bn.js';
+import React, { useCallback, useState } from 'react';
+
+import { InputAddress, InputNumber, Modal, TxButton } from '@polkadot/react-components';
+import { useApi, useCollectiveInstance, useCollectiveMembers } from '@polkadot/react-hooks';
+import { BN_ZERO } from '@polkadot/util';
+
+import { useTranslation } from '../translate';
+
+interface Props {
+  closeModal: () => void,
+  hash: string;
+}
+
+interface Threshold {
+  isThresholdValid: boolean;
+  threshold?: BN;
+}
+
+function SlashTipModal ({ closeModal, hash }: Props): React.ReactElement<Props> | null {
+  const [accountId, setAccountId] = useState<string | null>(null);
+  const [{ isThresholdValid, threshold }, setThreshold] = useState<Threshold>({ isThresholdValid: false });
```

Comment:
We should get the default from here - 

https://github.com/polkadot-js/apps/blob/master/packages/apps-config/src/api/params/proposalThresholds.ts

It is the same as the normal RejectOrigin threshold, not sure it is 100% clear in the above linked file, aka we could clarify/add.